force-app/main/default/pages/JobApplicationList.page

```diff
@@ -0,0 +1,15 @@
+<apex:page standardController="Job_Application__c" recordSetVar="applications">
+    <apex:pageBlock title="All Job Applications">
```

Comment:
Good to go. However, could we try using labels instead?
https://help.salesforce.com/s/articleView?id=sf.cl_about.htm&type=5

Reply:
Also try to apply labels for every other place in VF page